tests/robot-tests/.env.example

```diff
@@ -1,17 +1,18 @@
-PUBLIC_URL=http://localhost:3000  # (string) URL of the public frontend
-DATA_API_URL=http://localhost:5000/api  # (string) URL for the Data API
-CONTENT_API_URL=http://localhost:5010/api  # (string) URL for the Content API
-PUBLIC_AUTH_USER= # (string) the basic auth user for the public frontend (leave empty if none)
-PUBLIC_AUTH_PASSWORD= # (string) the basic auth password for the public frontend (leave empty if none)
-ADMIN_URL=https://localhost:5021  # (string) URL for the Admin frontend
-ADMIN_EMAIL=ADMIN_EMAIL  # (string) BAU1 user email
-ADMIN_PASSWORD=ADMIN_PASSWORD  # (string) BAU1 user password
-ANALYST_EMAIL=ANALYST_EMAIL  # (string) Analyst1 user email
-ANALYST_PASSWORD=ANALYST_PASSWORD  # (string) Analyst1 user password
-RELEASE_COMPLETE_WAIT=120  # (int) Time to wait for a release to publish
-WAIT_MEDIUM=120  # (int) Variable used throughout tests
-WAIT_SMALL=45 # (int) Variable used throughout tests
-WAIT_LONG=180  # (int) Variable used throughout tests
-TIMEOUT=30 # (int) Variable used throughout tests
-IMPLICIT_WAIT=15 # (int) Variable used throughout tests
+PUBLIC_URL=http://localhost:3000  # URL of the public frontend
+DATA_API_URL=http://localhost:5000/api  # URL for the Data API
+CONTENT_API_URL=http://localhost:5010/api  # URL for the Content API
+PUBLIC_AUTH_USER= # the basic auth user for the public frontend (leave empty if none)
+PUBLIC_AUTH_PASSWORD= # the basic auth password for the public frontend (leave empty if none)
+ADMIN_URL=https://localhost:5021  # URL for the Admin frontend
+ADMIN_EMAIL=ADMIN_EMAIL  # BAU1 user email
+ADMIN_PASSWORD=ADMIN_PASSWORD  # BAU1 user password
+ANALYST_EMAIL=ANALYST_EMAIL  # Analyst1 user email
+ANALYST_PASSWORD=ANALYST_PASSWORD  # Analyst1 user password
+RELEASE_COMPLETE_WAIT=120  # Time to wait for a release to publish
+WAIT_MEDIUM=120  # Variable used throughout tests
+WAIT_SMALL=45 # Variable used throughout tests
+WAIT_LONG=180  # Variable used throughout tests
+TIMEOUT=30 # Variable used throughout tests
+IMPLICIT_WAIT=15 # Variable used throughout tests
 FAIL_TEST_SUITES_FAST=1   # ("0" or "1") Fails entire test suite after any failure
+IDENTITY_PROVIDER= # ("AZURE" or "KEYCLOAK") opt into using either a Keycloak or Azure based identity provider
```

Comment:
We normally provide defaults in this example file - wondering if we should just add AZURE here as a default?